terraform/ec2/README.md

```diff
@@ -169,6 +169,7 @@ Outputs:
         "s3:GetObject",
         "s3:GetObjectAcl",
         "s3:PutObject",
+        "s3:DeleteObject",
```

Comment:
Why we need deletion for s3 though?

Reply:
Delete old canary instance. See other pr for private repo.